build

```diff
@@ -14,7 +14,9 @@ export GOPATH=${PWD}/gopath
 eval $(go env)
 
 echo "Building rkt (stage0)..."
-go build -o $GOBIN/rkt ${REPO_PATH}/rkt
+eval go build -o ${GOBIN}/rkt \
+	${RKT_STAGE1_IMAGE:+-ldflags \'-X main.defaultStage1Image "${RKT_STAGE1_IMAGE}"\'} \
```

Comment:
You are escaping ' but not inside of another string. You probably want:

    ${RKT_STAGE1_IMAGE:+-ldflags "-X main.defaultStage1Image '${RKT_STAGE1_IMAGE}'"} \

And drop the eval. It is only going to make escaping more complicated.

Alternatively it may be clearer to construct arguments in an array:

```sh
args=( build -o "${GOBIN}/rkt" )
if [[ -n "${RKT_STAGE1_IMAGE}" ]]; then
    args+=( -ldflags "-X main.defaultStage1Image '${RKT_STAGE1_IMAGE}'" )
fi
go "${args[@]}" "${REPO_PATH}/rkt"
```

or something like that.